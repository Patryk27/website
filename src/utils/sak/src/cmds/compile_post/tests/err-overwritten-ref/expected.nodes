Element(
    Spanned {
        inner: Element {
            name: Spanned {
                inner: "ref",
                span: Span {
                    beg: 1,
                    end: 3,
                },
            },
            attrs: [
                Attr {
                    name: Spanned {
                        inner: "id",
                        span: Span {
                            beg: 5,
                            end: 6,
                        },
                    },
                    value: Some(
                        Spanned {
                            inner: "a",
                            span: Span {
                                beg: 8,
                                end: 10,
                            },
                        },
                    ),
                },
            ],
            children: [
                Text(
                    Spanned {
                        inner: "\n    http://a.com\n",
                        span: Span {
                            beg: 12,
                            end: 29,
                        },
                    },
                ),
            ],
        },
        span: Span {
            beg: 0,
            end: 35,
        },
    },
)
Text(
    Spanned {
        inner: "\n\n",
        span: Span {
            beg: 36,
            end: 37,
        },
    },
)
Element(
    Spanned {
        inner: Element {
            name: Spanned {
                inner: "ref",
                span: Span {
                    beg: 39,
                    end: 41,
                },
            },
            attrs: [
                Attr {
                    name: Spanned {
                        inner: "id",
                        span: Span {
                            beg: 43,
                            end: 44,
                        },
                    },
                    value: Some(
                        Spanned {
                            inner: "a",
                            span: Span {
                                beg: 46,
                                end: 48,
                            },
                        },
                    ),
                },
            ],
            children: [
                Text(
                    Spanned {
                        inner: "\n    http://b.com\n",
                        span: Span {
                            beg: 50,
                            end: 67,
                        },
                    },
                ),
            ],
        },
        span: Span {
            beg: 38,
            end: 73,
        },
    },
)