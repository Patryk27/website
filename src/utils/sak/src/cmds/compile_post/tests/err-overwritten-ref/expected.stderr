[31merror:[0m ref overwrites another ref
   â•­â”€[input:5:9]
   â”‚
 1 â”‚ <ref id="a">
   â”‚         â”€â”¬â”€  
   â”‚          â•°â”€â”€â”€ ref defined here before
   â”‚ 
 5 â”‚ <ref id="a">
   â”‚         â”€â”¬â”€  
   â”‚          â•°â”€â”€â”€ ref overwrites another ref
â”€â”€â”€â•¯

compilation failed
